### PR TITLE
Add changelog for v1.3

### PR DESCRIPTION
**Pull request by xing-yang**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
This PR adds changelog for release 1.3.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

### CODE REVIEW

File: CHANGELOG-1.3.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Changelog since v1.2.0
+
+## Notable Changes
+
+- Snapshotter will no longer call ListSnapshots if the CSI Driver does not support this operation. ([#138](https://github.com/kubernetes-csi/external-snapshotter/pull/138), [@ggriffiths](https://github.com/ggriffiths))
+
+- Add a prefix "external-snapshotter-leader" and the driver name to the snapshotter leader election lock name. ([#129](https://github.com/kubernetes-csi/external-snapshotter/pull/129), [@zhucan](https://github.com/zhucan))
```
Comment (msau42) [inline CHANGELOG-1.3.md]: Is this a potentially breaking change and wouldn't work with upgrades?

Comment (xing-yang) [reply CHANGELOG-1.3.md]: It probably is.  The controller won't recognize old lock names any more.  Let me move this to a breaking change section.

Comment (msau42) [reply CHANGELOG-1.3.md]: Can you also add some more description on what won't work (ie HA upgrades), any possible problems that could result (such as multiple snapshots created), and any possible mitigations?

Comment (xing-yang) [reply CHANGELOG-1.3.md]: Updated.  PTAL.

**Pull request merged**